### PR TITLE
edit button fixed

### PR DESCRIPTION
**Pull request by Maheima**
@ankushdharkar I have solved #222

### CODE REVIEW

File: src/components/navBar/index.tsx [hunk 1]
```diff
@@ -1,17 +1,68 @@
...
+      //alt key's keycode is 18
+      if(event.keyCode == 18){
+        setIsEditVisible(true);
+      }
+    }
+    document.addEventListener('keydown', debounce(handleKeyDown, 300));
+    
+      return function cleanup() {
+      document.removeEventListener('keydown', handleKeyDown);
+    }
```
Comment (ankushdharkar) [inline src/components/navBar/index.tsx:45]: Should this be a custom hook?

File: src/components/navBar/index.tsx [hunk 2]
```diff
@@ -1,17 +1,68 @@
 import Image from 'next/image';
 import styles from '@/components/navBar/navBar.module.scss';
-
+import { useEffect, useState } from 'react';
+import Link from 'next/link';
 const RDSLogo = '/RDSLogo.png';
+const edit = '/pencil.webp';
+
+function debounce<A = unknown, R = void>(
```
Comment (ankushdharkar) [inline src/components/navBar/index.tsx:8]: Why is this `unknown`?

File: src/components/navBar/index.tsx [hunk 3]
```diff
@@ -1,17 +1,68 @@
...
+  fn: (args: A) => R,
+  ms: number
+): (args: A) => Promise<R> {
+  let timer: NodeJS.Timeout;
```
Comment (ankushdharkar) [inline src/components/navBar/index.tsx:12]: Should we have a `NodeJS` type in NextJS?

File: src/components/navBar/index.tsx [hunk 4]
```diff
@@ -1,17 +1,68 @@
...
+  const [isEditVisible, setIsEditVisible] = useState(false);
+
+  
+
+  useEffect(() => {
+    function handleKeyDown(event:any) {
+      //alt key's keycode is 18
+      if(event.keyCode == 18){
+        setIsEditVisible(true);
+      }
```
Comment (ankushdharkar) [inline src/components/navBar/index.tsx:39]:
```suggestion
      if(event.keyCode == ALT_KEY){
        setIsEditVisible(true);
      }
```

Comment (ankushdharkar) [inline src/components/navBar/index.tsx:39]:
```suggestion
      if(event.keyCode == ALT_KEY){
        setIsEditVisible(true);
      }
```

File: src/components/navBar/index.tsx (hunk 3 above)
Comment (Maheima) [reply src/components/navBar/index.tsx:12]: I couldn't find anything specific to nextjs in timer type

**Comment by Maheima** [comment]
The further updates to this PR are done at this new PR https://github.com/Real-Dev-Squad/website-status/pull/278

File: src/components/navBar/index.tsx (hunk 1 above)
Comment (Maheima) [reply src/components/navBar/index.tsx:45]: https://github.com/Real-Dev-Squad/website-status/pull/278 fixed it in this PR

**Comment by Maheima** [comment]
working on it on this new PR https://github.com/Real-Dev-Squad/website-status/pull/278

File: src/components/navBar/index.tsx (hunk 2 above)
Comment (Maheima) [reply src/components/navBar/index.tsx:8]: made a custom hook and changed its type here https://github.com/Real-Dev-Squad/website-status/pull/278

File: src/components/navBar/index.tsx (hunk 1 above)
Comment (Maheima) [reply src/components/navBar/index.tsx:45]: https://github.com/Real-Dev-Squad/website-status/pull/278 done it in this PR

**Pull request closed without merging**